src/app/documentation/documentation.component.html

```diff
@@ -37,7 +37,6 @@ <h2>PDF Documentation</h2>
                       <div class="btn-group-vertical btn-block" role="group">
                         <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/OutputDetailsAndExamples.pdf" class="btn btn-body border text-left" type="button">Output Details And Examples</a>
                         <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/PlantApplicationGuide.pdf" class="btn btn-body border text-left" type="button">Plant Application Guide</a>
-                        <a href="assets/nrel_custom/pdfs/pdfs_v{{latestRelease.version}}/TipsAndTricksUsingEnergyPlus.pdf" class="btn btn-body border text-left" type="button">Tips and Tricks Using EnergyPlus</a>
```

Comment:
We don't have the tips and tricks in the install anymore.  It's on [RTD](https://energyplus.readthedocs.io/en/latest/tips_and_tricks/tips_and_tricks.html) now, where it belongs.